corehq/apps/user_importer/migrations/0001_initial.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-06-30 02:39
+from __future__ import unicode_literals
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='UserUploadRecord',
```

Comment:
nit: UserUploadLog?

Reply:
This naming convention is what we use for most similar models including `CaseUploadRecord`. I like it a bit better, and its not really a log, which I think of as a big text file you append to, although thats obviously a bit reductive